Client/Frontend/Strings.swift

```diff
@@ -1261,8 +1261,8 @@ extension String {
                 public static let ThoughtProvokingStoriesSubtitle = MZLocalizedString(
                     key: "Settings.Home.Option.ThoughtProvokingStories.subtitle.v116",
                     tableName: "CustomizeFirefoxHome",
-                    value: "Articles powered by Pocket",
-                    comment: "In the settings menu, in the Firefox homepage customization section, this is the subtitle for the option that allows users to turn the Pocket Recommendations section on the Firefox homepage on or off")
+                    value: "Articles powered by %@",
```

Comment:
If the value is changed we should change the key, unless that key isn't exported yet?

Reply:
It's not imported yet. @Delphine pinged me in the import PR to make the changes.